Nodejs/Product/Nodejs/Snippets/1033/Nodejs/switch.snippet

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>switch</Title>
+      <Description>Code Snippet for a Switch Statement</Description>
+      <Shortcut>switch</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>true</ID>
+          <Default>true</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[switch ($true$) {

```

Comment:
In this case there's no value to supplying something like true/false, so I'd just go with ```switch_on```, which is consistent with the default javascript editing experience in VS